framework/validators/ExistValidator.php

```diff
@@ -92,7 +92,7 @@ public function validateAttribute($model, $attribute)
     {
         $targetAttribute = $this->targetAttribute === null ? $attribute : $this->targetAttribute;
         $params = $this->prepareConditions($targetAttribute, $model, $attribute);
-        $conditions[] = $this->targetAttributeJunction == 'or' ? 'or' : 'and';
+        $conditions = [ $this->targetAttributeJunction == 'or' ? 'or' : 'and' ];
```

Comment:
Why?

Reply:
Just want to fix [scrunitizer issue](https://scrutinizer-ci.com/g/yiisoft/yii2/issues/master/files/framework/validators/ExistValidator.php?selectedLabels%5B0%5D=3&selectedLabels%5B1%5D=10&orderField=path&order=asc&honorSelectedPaths=0)

Reply:
Ah. Right. Looks good. One thing is that `[` and `]` should not have extra spaces.

Reply:
Extra spaces removed